app/views/families/show.html.erb

```diff
@@ -0,0 +1,2 @@
+<h1><%= I18n.t("families.show.title") %></h1>
```

Comment:
I believe you don't need to call the `I18n` here. And also just use `t(".title")`

Reply:
👌